[0.019s] Invoking command in '/home/sann/ros_ws/build/custom_interfaces': CONDA_PROMPT_MODIFIER=(base) /home/sann/anaconda3/bin/cmake /home/sann/ros_ws/src/custom_interfaces -DCMAKE_INSTALL_PREFIX=/home/sann/ros_ws/install/custom_interfaces
[0.175s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.175s] -- Found ament_cmake_core: 1.3.11 (/opt/ros/humble/share/ament_cmake_core/cmake)
[0.219s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.221s] -- Found rosidl_adapter: 3.1.6 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.249s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.270s] -- Found action_msgs: 1.2.1 (/opt/ros/humble/share/action_msgs/cmake)
[0.298s] -- Found geometry_msgs: 4.2.4 (/opt/ros/humble/share/geometry_msgs/cmake)
[0.460s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.460s]   execute_process(/home/sann/anaconda3/bin/python3 -m rosidl_adapter
[0.460s]   --package-name custom_interfaces --arguments-file
[0.460s]   /home/sann/ros_ws/build/custom_interfaces/rosidl_adapter__arguments__custom_interfaces.json
[0.460s]   --output-dir
[0.460s]   /home/sann/ros_ws/build/custom_interfaces/rosidl_adapter/custom_interfaces
[0.460s]   --output-file
[0.460s]   /home/sann/ros_ws/build/custom_interfaces/rosidl_adapter/custom_interfaces.idls)
[0.460s]   returned error code 1:
[0.460s] 
[0.460s]   Traceback (most recent call last):
[0.460s] 
[0.460s]     File "<frozen runpy>", line 198, in _run_module_as_main
[0.460s]     File "<frozen runpy>", line 88, in _run_code
[0.461s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.461s]       sys.exit(main())
[0.461s]                ^^^^^^
[0.461s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.461s]       abs_idl_file = convert_to_idl(
[0.461s]                      ^^^^^^^^^^^^^^^
[0.461s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 23, in convert_to_idl
[0.461s]       from rosidl_adapter.srv import convert_srv_to_idl
[0.461s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/srv/__init__.py", line 16, in <module>
[0.461s]       from rosidl_adapter.resource import expand_template
[0.461s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/resource/__init__.py", line 19, in <module>
[0.461s]       import em
[0.461s] 
[0.461s]   ModuleNotFoundError: No module named 'em'
[0.461s] 
[0.461s] Call Stack (most recent call first):
[0.461s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.461s]   CMakeLists.txt:12 (rosidl_generate_interfaces)
[0.461s] 
[0.461s] [0m
[0.461s] -- Configuring incomplete, errors occurred!
